packages/grid/_modules_/grid/components/panel/PanelHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { classnames } from '../../utils/classnames';
+
+const useStyles = makeStyles(
+  () => ({
+    root: {
+      padding: 8,
```

Comment:
I wonder if it should be 
```suggestion
      padding: theme.spacing(1),
```